partcad/src/partcad/shape.py

```
@@ -450,6 +450,38 @@
     ):
         asyncio.run(self.render_step_async(ctx, project, filepath))
 
+    async def render_brep_async(
```

Ruff (0.8.2)

Missing return type annotation for public function `render_brep_async`

Add return type annotation: `None`

(ANN201)